easybuild/easyconfigs/c/CP2K/CP2K-9.1-CrayGNU-cuda.psmp

```diff
@@ -13,13 +13,14 @@ AR             = ar -r
 
 # LIBXSMM (-D__LIBXSMM) is slower than LIBSMM (-D__HAS_smm_dnn)
 DFLAGS   = -D__ACC -D__DBCSR_ACC -D__GRID_CUDA -D__GFORTRAN -D__HAS_smm_dnn -D__MAX_CONTR=4 -D__MPI_VERSION=3 -D__parallel -D__SCALAPACK
-DFLAGS  += -D__ELPA -D__ELPA_NVIDIA_GPU -D__FFTW3 -D__GSL -D__LIBINT -D__LIBVORI -D__LIBXC -D__PLUMED2 -D__SPGLIB
+DFLAGS  += -D__ELPA -D__ELPA_NVIDIA_GPU -D__FFTW3 -D__GSL -D__LIBINT -D__LIBVORI -D__LIBXC -D__PLUMED2 -D__SIRIUS -D__SPGLIB
```

Comment:
you need -D__GRID_CUDA and possibly -D__PW_CUDA to get more out of the GPU. some parts of the code are now offloaded to the GPU. these two options will enable them.